public/github.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-github" viewBox="0 0 16 16">
```

Comment:
you added this whilst I was writing the review on the GitHub logo not working, what was the reason for an svg file? I know they're commonly used for quality reasons, but im sure a jpeg or a png is alright too? maybe im just lazy and not fancy like you guys.